src/types/events-types.ts

```diff
@@ -23,10 +24,12 @@ export type EventsRecordsFilter = {
   dateCreated?: RangeCriterion;
 };
 
-export type EventsQueryFilter = EventsMessageFilter | EventsRecordsFilter | ProtocolsQueryFilter;
+// a union type of the different types of filters a user can use when issuing an EventsQuery or EventsSubscribe
```

Comment:
Use `/**` syntax so SDK doc-gen tool and IDE intelli-sense can pick it up. Same goes to two type comments above.